generators/app/aspnetHelper.js

```diff
@@ -91,46 +117,108 @@ AspNetHelper.prototype.configureUrls = function (cb) {
                     return;
                 });
                 return;
-            }
-            cb(null, null);
+            });
             return;
-        });
-    } else {
-        var rootFolder = process.cwd() + path.sep;
-        var fileName = rootFolder + 'project.json';
-        var backupFile = rootFolder + 'project.json.backup';
-        var port = this._portNumber;
-        var self = this;
-        fs.readFile(fileName, 'utf8', function (err, data) {
-            if (err) {
-                cb(new Error('Can\'t read project.json file. Make sure project.json file exists.'));
-                return;
-            }
+        }
+        cb(null, null);
+        return;
+    });
+}
 
-            // Remove BOM.
-            if (data.charCodeAt(0) === 0xFEFF) {
-                data = data.replace(/^\uFEFF/, '');
-            }
+/**
+ * RC1: Checks if  'web' command is in the  project.json and adds it if command is not there yet.
+ * RC2: Ensures buildOptions is using debugType portable and publishOptions includes the dockerfiles
+ * @returns {string}
+ */
+AspNetHelper.prototype.updateProjectJson = function (cb) {
+    var rootFolder = process.cwd() + path.sep;
+    var fileName = rootFolder + 'project.json';
+    var backupFile = rootFolder + 'project.json.backup';
+    var port = this._portNumber;
+    var dotnetVersion = this._dotnetVersion;
+    var self = this;
+    fs.readFile(fileName, 'utf8', function (err, data) {
+        if (err) {
+            cb(new Error('Can\'t read project.json file. Make sure project.json file exists.'));
+            return;
+        }
 
-            data = JSON.parse(data);
+        // Remove BOM.
+        if (data.charCodeAt(0) === 0xFEFF) {
+            data = data.replace(/^\uFEFF/, '');
+        }
+
+        data = JSON.parse(data);
+
+        if (dotnetVersion === 'RC2' ) {
+            var changed = false;
+            if (data.buildOptions === undefined) {
```

Comment:
nit: you could replace `data.buildOptions === undefined` with `!data.buildOptions`